CONTRIBUTING.md

```diff
@@ -94,6 +94,11 @@ Any time changes are made to the GO code, you will have to stop the server and r
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`. One thing to note, you might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
 After running Meshery server, you will need to select your **Cloud Provider** by navigating to `localhost:9081`. Only then you will be able to use the Meshery UI on port `3000`.
 
+### Common Issues
+
+#### Invalid C headers ( Mac OS )
```

Comment:
I would have wanted this to be added in the last section under the heading notes rather than in between the installation guide.